### ISSUE TITLE
Date selection not disabled on readonly input

### ISSUE THREAD

**Issue by jnthnjns** [dbushell/Pikaday#662]
I would like to request that when an input has the property `readonly` the logic should be to either not trigger the date selection or to show the date selection but disable any on selection triggers.

Example:
```html
<input type="text" id="CompletedDate" name="CompletedDate" type="datetime" readonly />
```

In this instance the date is changeable and is submitted with the new value.

[readonly - HTML 4.01](https://www.w3.org/TR/html401/interact/forms.html#adef-readonly)
[readonly - HTML 5](https://www.w3.org/TR/html5/forms.html#the-readonly-attribute)

**Comment by terryroe** [comment]
What is the status of this issue?  Will the fix be accepted soon?

**Comment by claar** [comment]
Also needing this.

**Comment by tpetrone** [comment]
I agree with @mboudreau `readonly` should avoid user from typing instead of actual picking the date from the calendar. Disabling it prevents the user from interacting with the component at all.

**Comment by rodolfojnn** [comment]
This work for me: In onOpen callback, set: if (htmlElement.readOnly) { return picker.hide(); }
